relayer/client-tx.go

```diff
@@ -349,3 +349,94 @@ func IsMatchingClient(clientStateA, clientStateB ibctmtypes.ClientState) bool {
 func IsMatchingConsensusState(consensusStateA, consensusStateB *ibctmtypes.ConsensusState) bool {
 	return reflect.DeepEqual(*consensusStateA, *consensusStateB)
 }
+
+// GetClientAndUpdate update clients to prevent expiry
+func GetClientAndUpdate(src, dst *Chain, thresholdTime time.Duration) (time.Duration, error) {
```

Comment:
this name is misleading in some way, this doesn't return the client but the trusting period.